libs/pageserver_api/src/models.rs

```diff
@@ -309,6 +310,33 @@ pub struct EvictionPolicyLayerAccessThreshold {
     pub threshold: Duration,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+pub struct ThrottleConfig {
+    pub task_kinds: Vec<String>, // TaskKind
+    pub initial: usize,
+    pub interval_millis: NonZeroU64,
+    pub interval_refill: NonZeroUsize,
```

Comment:
```suggestion
    /// Milliseconds
    pub interval_refill: NonZeroUsize,
```

Or even better: move the duration conversion from `pageserver/..../throttle.rs` as a method on this type `fn refill_every(&self) -> Duration`.

Reply:
Hehe, I got confused already ofc the millis is the `internal_millis`, still, I think the duration conversion should be a method on this type.